lib/new.js

```diff
@@ -43,6 +43,20 @@ function getPackageName(name) {
   return `skyux-${prefix}-${name}`;
 }
 
+/**
+ * Returns a string to be used for the description property in package.json/logger.
+ * First letter is capitalized - so expected to be used first in a sentence.
```

Comment:
Probably want to delete this line since we shifted to use a lowercase first letter.